### DISCUSSION TITLE
Simultaneous USB webcam and SD card recording

### DISCUSSION THREAD

**Discussion by kmeine** [gopro/labs#208] [Feature Requests]
I get that wired applications are kinda bottom of the barrel, but I have an application where some stuff needs to get data over wires, period. It would be really nice if the USB video, which can be super janky because it's just being used to let a human make some decisions, didn't shut off the high quality record on the SD card. Seems a bit silly that it can wireless stream and record simultaneously but not replicate that functionality wired, and I don't wanna have to switch to Mevos to get it. Can someone just flip a bit in there?

**Comment by dnewman-gpsw** [comment]
Yes, it is good idea, and I've personally requested it. Unfortunately it is beyond a straight forward Labs hack. Labs never modifies the video pipeline, as reliable captures is still a Labs priority. My only guess to why it was done this way is a wired camera is typically a stationary camera, and running two encoders and the SD card, is more heat to worry about. And 98% of webcam users, aren't seeking to have a high quality local copy (yet I think it would be useful.)
